docs/website/packages/codegen/generate.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: Generating
+description: codegen's generate function
+---
+
+The generate function loads FHIR structure definition files, and integrates them into typescript code, which is determined by templates that you prepare. These templates are in [EJS format](https://ejs.co/). The files that generate the definitions can be found at `packages/codegen/definitions/fhir`.
```

Comment:
I am not a fan of "and integrates them into typescript code"; I don't think integrate is the proper word.
Also - the generator is agnostic to the target language; it does not need to be typescript, it can be anything.